libherokubuildpack/src/download.rs

```diff
@@ -2,15 +2,18 @@ use std::{fs, io};
 
 #[derive(thiserror::Error, Debug)]
 pub enum DownloadError {
-    // Boxed to prevent `large_enum_variant` errors since `ureq::Error` is massive.
```

Comment:
Note: no longer necessary since `reqwest` wraps their errors inside `reqwest::Error` in a `Box` already.